### PR TITLE
fixed mobile ai page

### PR DESCRIPTION
**Pull request by a-wong-8**

### CODE REVIEW

File: src/pages/ai.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function AiPage() {
                 <p className="text-xl">{heroSection.text}</p>
                 <div>
                   <Link
-                    className="rounded-full bg-gradient-to-b from-green-500 to-blue-500 px-10 py-6 text-2xl font-medium lg:py-4 lg:px-8 lg:text-lg"
+                    className="rounded-full bg-gradient-to-b from-green-500 to-blue-500 px-10 py-6 text-2xl font-medium lg:py-4 lg:px-8 lg:text-lg text-sm"
```
Comment (gevuong) [inline src/pages/ai.tsx]: there's a css conflict here, `text-sm` and `text-2xl` are set. Do you get a css conflict warning in your workspace? 

Also as a side note, it looks like the auto class sorting with prettier isn't taking effect. For example, `text-sm` remains last in the list of class names. It should be where `text-2xl` is if auto class sorting worked.

File: src/pages/ai.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ export default function AiPage() {
       </section>
 
       {/* Gradient Overlay Container */}
-      <div className="relative -mb-24 bg-gradient-to-b from-[#162b4c] to-[#3468b2]">
+      <div className="relative -mb-24 bg-gradient-to-b from-[#162b4c] to-[#3468b2] ">
```
Comment (gevuong) [inline src/pages/ai.tsx]: remove trailing whitespace

File: src/pages/ai.tsx (hunk 1 above)
Comment (a-wong-8) [reply src/pages/ai.tsx]: where would I see this error?

Comment (gevuong) [reply src/pages/ai.tsx]: In your workspace. A squiggly line would appear underneath `text-sm` and `text-2xl`.
Screenshot 2024-05-28 at 10 38 01 AM

Comment (a-wong-8) [reply src/pages/ai.tsx]: hmm I dont see that on mine

Comment (a-wong-8) [reply src/pages/ai.tsx]: I fixed it on current push

Comment (gevuong) [reply src/pages/ai.tsx]: is your prettier tailwind plugin working?

Comment (a-wong-8) [reply src/pages/ai.tsx]: do you mean extension?

Comment (gevuong) [reply src/pages/ai.tsx]: I think you need both prettier tailwind plugin and tailwind css intellisense enabled in order for tailwind linting and auto class sorting to properly work

**Review by gevuong — APPROVED** [review]

File: src/pages/ai.tsx (hunk 1 above)
Comment (a-wong-8) [reply src/pages/ai.tsx]: thank capt. do you mean Tailwind class sorter? I cant find the prettier tailwind plugin

Comment (gevuong) [reply src/pages/ai.tsx]: Slack me to discuss further

**Pull request merged**